build.sh

```diff
@@ -87,7 +87,7 @@ if [ ${BUILD_LINUX} -gt 0 ]; then
     }
     popd
 
-    LIBS="inst/lib/*.so JVips/src/main/c/libJVips.so"
```

Comment:
Are you sure it doesn't exist? I basically copy-pasted it from resources declared in pom.xml.